src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <label class="form-field">
+            Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
+        </label>
+        <label class="form-field">
+            Name:
+          <input
+           type="text"
+           name="name"
+           autocomplete="off"
+           required
+          >
+        </label>
+        <label class="form-field">
+            How old are You?
+          <input
+           type="number"
+           name="age"
+           min="1"
+           max="100"
+           value="12"
+           required
+          >
+        </label>
+        <label class="form-field">
+            Full date of birth:
+          <input
+           type="date"
+           name="dateOfbirth"
+           required
+          >
+        </label>
+        <label class="form-field">
+              I accept the term of the agreement
+          <input
+          type="checkbox"
+          name="agree"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>Registration</legend>
+        <label class="form-field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com"
+          required
+          >
+        </label>
+        <label class="form-field">
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="8"
+          maxlength="25"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="container">
+        <legend>An interesting fact about you!</legend>
+        <label class="form-field">
+          Do you love cats?
+          <input type="radio" name="cats">Yes
+          <input type="radio" name="cats">No
+        </label>
+        <label class="form-field">
+          What is your favorite color?
```

Comment:
```suggestion
      <fieldset class="container">
        <legend>An interesting fact about you!</legend>
        
        <label class="form-field">
          Do you love cats?
          <input type="radio" name="cats">Yes
          <input type="radio" name="cats">No
        </label>
        
        <label class="form-field">
          What is your favorite color?
```
Add newlines between adjacent elements to improve code readability. 
Also, please pay attention to these rule in "Do you love cats?"
![image](https://user-images.githubusercontent.com/96409563/209790965-b4e9b66a-5b3d-44b4-b12a-d4af4b4a2c04.png)

Reply:
not fixed, if you have any question - feel free to ask them in fe_chat